data/templates/install-coreos.sh

```diff
@@ -3,5 +3,11 @@ set -e
 CLOUD_CONFIG_FILE=pxe-cloud-config.yml
 curl -o $CLOUD_CONFIG_FILE http://<%=server%>:<%=port%>/api/current/templates/$CLOUD_CONFIG_FILE?nodeId=<%=nodeId%>
 sudo coreos-install -d <%=installDisk%> -c $CLOUD_CONFIG_FILE -b <%=repo%>
+
+# Customizations for supporting CoreOS Ignition:
+mkdir /mnt/coreos
+mount <%=installDisk%>6 /mnt/coreos/
```

Comment:
Hello @yyscamper,
The number `6` is required, although I can see how it may look like a typo.
Please see [the CoreOS partition table](https://coreos.com/os/docs/latest/sdk-disk-partitions.html#partition-table). The change to `grub.cfg` that we make then gets picked up by CoreOS at first boot. Let me know if I can offer any more clarification.
Thanks!
Tim